styles/main.css

```diff
@@ -151,11 +157,14 @@ header li a {
 
 header li a:hover{
     border-top: 4px solid hotpink;
+    text-decoration: none;
```

Comment:
You need to override the default underlining in these more specific hyperlink selectors